machine-check/scripts/test/cjs-copy.ts

```diff
@@ -0,0 +1,22 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { globSync } from 'glob'
+import { TEST_CJS_DIR, TEST_ESM_DIR, TEST_ESM_GLOB } from './cjs-common.js'
+
+const createNotice = ({ sourcePath }: { sourcePath: string }) =>
+  `
+// Do not edit! This file is autogenerated from
+// ${sourcePath} 
+`.trim() + '\n\n'
+
+globSync(TEST_ESM_GLOB, {
+  ignore: ['**/jest.config.ts'],
+}).forEach((sourcePath) => {
+  const destinationPath = path.resolve(TEST_CJS_DIR, path.relative(TEST_ESM_DIR, sourcePath))
+
+  let content = fs.readFileSync(sourcePath, 'utf8')
+  content = content.replace(/from '(.*).js'/gi, (_, p1) => `from '${p1}'`)
```

Comment:
We don’t have dynamic imports, right? (I just remembered that this was the problem with konva, so we might also patch `import('...')` syntax proactively) Or is it guaranteed that running the tests will catch such an issue?

Reply:
https://github.com/konvajs/konva/pull/1514/files